input/fsh/Terminology.fsh

```diff
@@ -109,7 +196,7 @@ Description: "Valueset of the kind of treatment a patient underwent"
 * include TreatmentTypesCodeSystem#6 "Hormonal therapy"
 * include TreatmentTypesCodeSystem#7 "Targeted therapy"
 * include TreatmentTypesCodeSystem#8 "Best supportive care"
-* include TreatmentTypesCodeSystem#999 "Unknown"
+* include NullFlavor#UNK "unknown"
```

Comment:
Lets go with #ASKU "asked but unknown" for these, it is more accurate and would make more sense. It'll also fit into Observations better.